app/models/user.rb

```diff
@@ -119,7 +119,7 @@ def no_attempt_for_two_weeks
       if volunteer.supervisor.active? &&
           volunteer.case_assignments.any? { |assignment| assignment.active? } &&
           (volunteer.case_contacts.none? ||
-          volunteer.case_contacts.maximum(:occurred_at) < (Time.zone.now - 14.days))
```

Comment:
Since `occurred_at` is not required for a draft, if a user has only drafts with `occurred_at` set to nil, this creates a `NoMethodError`.